modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1123,23 +1123,44 @@
 	body_parts_covered = CHEST|LEGS|ARMS
 	mutant_variants = NONE
 
+//Donation reward for Hacker T.Dog
 /obj/item/clothing/head/nanotrasen_representative/hubert
 	name = "CC Ensign's cap"
 	desc = "A tailor made peaked cap, denoting the rank of Ensign."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
 	icon_state = "CCofficerhat"
 
+//Donation reward for Hacker T.Dog
 /obj/item/clothing/suit/armor/vest/nanotrasen_representative/hubert
 	name = "CC Ensign's armoured vest"
 	desc = "A tailor made Ensign's armoured vest, providing the same protection - but in a more stylish fashion."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/suit.dmi'
 	icon_state = "CCvest"
 
+//Donation reward for Hacker T.Dog
 /obj/item/clothing/under/rank/nanotrasen_representative/hubert
 	name = "CC Ensign's uniform"
 	desc = "A tailor-made Ensign's uniform, various medals and chains hang down from it."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/uniform.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform.dmi'
 	icon_state = "CCofficer"
+
+//Donation reward for Cherno_00
+/obj/item/clothing/head/ushanka/frosty
+	name = "blue ushanka"
+	desc = "A dark blue ushanka with a hand stitched snowflake on the front. Cool the the touch."
```

Comment:
```suggestion
	desc = "A dark blue ushanka with a hand-stitched snowflake on the front. Cool to the touch."
```

Reply:
Good catch with the "the the". I copypasted it from the donator's suggestion and didn't notice.